packages/iterator-utilities/src/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"extends": "../../../tsconfig.base.json",
 	"compilerOptions": {
-		"lib": ["dom", "esnext"],
+		"lib": ["esnext"],
```

Comment:
As I said, you can then drop the `lib` field entirely, `esnext` is the base set by @sapphire/ts-config